core/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestNodesStatsAction.java

```diff
@@ -48,9 +54,38 @@ public RestNodesStatsAction(Settings settings, RestController controller) {
         controller.registerHandler(GET, "/_nodes/stats/{metric}", this);
         controller.registerHandler(GET, "/_nodes/{nodeId}/stats/{metric}", this);
 
-        controller.registerHandler(GET, "/_nodes/stats/{metric}/{indexMetric}", this);
+        controller.registerHandler(GET, "/_nodes/stats/{metric}/{index_metric}", this);
 
-        controller.registerHandler(GET, "/_nodes/{nodeId}/stats/{metric}/{indexMetric}", this);
+        controller.registerHandler(GET, "/_nodes/{nodeId}/stats/{metric}/{index_metric}", this);
```

Comment:
We have no examples at all of retrieving nodes stats with an index metric, can we add some to https://www.elastic.co/guide/en/elasticsearch/reference/5.0/cluster-nodes-stats.html so that users are aware of the correct way to retrieve them?